Source/SwiftLintFramework/Rules/ForceCastRule.swift

```diff
@@ -26,8 +26,7 @@ public struct ForceCastRule: Rule {
         ruleName: "Force Cast Rule",
         ruleDescription: "This rule checks whether you don't do force casts.",
         nonTriggeringExamples: [
-            "NSNumber() as? Int\n",
```

Comment:
Because of `commentDoesntViolate` in [`verifyRule()`](https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFrameworkTests/TestHelpers.swift#L18).